src/lib/tradesHandler.ts

```diff
@@ -36,7 +40,7 @@ export const triggerBuyOrders = async () => {
       // Send order to alpaca:
       AlpacaTradePromises.push(
         alpacaService
-          .postOrder(ticker, Side.BUY, price, quantity)
+          .postNewBuyOrder(ticker, Side.BUY, price, quantity)
```

Comment:
since "postNewBuyOrder" implicy says "buy", the argument "Side.BUY" should be removed and applied within the "postNewBuyOrder"-function